keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -105,8 +112,17 @@ data class ClusterSpec(
   override val locations: SubnetAwareLocations,
   private val _defaults: ServerGroupSpec,
   @JsonInclude(NON_EMPTY)
-  val overrides: Map<String, ServerGroupSpec> = emptyMap()
-) : Monikered, Locatable<SubnetAwareLocations> {
+  val overrides: Map<String, ServerGroupSpec> = emptyMap(),
+  @JsonIgnore
+  override val deliveryArtifact: DeliveryArtifact? = null,
+  @JsonIgnore
+  override val artifactVersion: String? = null,
+  @JsonIgnore
+  override val maxDiffCount: Int? = 2,
+  @JsonIgnore
+  // Once clusters go unhappy, only retry when the diff changes, or if manually unvetoed
+  override val unhappyWaitTime: Duration? = Duration.ZERO
```

Comment:
This means that unhappy clusters remain paused indefinitely until either the diff changes, or a user intervenes.